conda-forge.yml

```diff
@@ -1,9 +1,14 @@
 azure:
   free_disk_space: true
+  store_build_artifacts: true
```

Comment:
Enabling this configuration stores build artifacts on CI that can be downloaded for local testing (without needing to rebuild locally)